sycl/unittests/spec_constants/DefaultValues.cpp

```diff
@@ -205,7 +205,7 @@ static sycl::unittest::PiImage generateDefaultImage() {
 sycl::unittest::PiImage Img = generateDefaultImage();
 sycl::unittest::PiImageArray ImgArray{Img};
 
-TEST(DefaultValues, DefaultValuesAreSet) {
+TEST(DefaultValues, DISABLED_DefaultValuesAreSet) {
```

Comment:
Shall we submit some issue to re-enable those tests? Or someone is already working on a fix?